test/processors/concerns/http_files_test.rb

```diff
@@ -23,6 +23,7 @@ def logger
 
   let(:http_uri) { URI.parse('http://test.prx.org/test/file.mp2') }
   let(:https_uri) { URI.parse('https://test.prx.org/test/file.mp2') }
+  let(:small_uri) { URI.parse('http://test.prx.org/test/small.txt') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.